tangelo/tangelo/server.py

```diff
@@ -541,6 +541,10 @@ def dirlisting(dirpath, reqpath):
         return result
 
     def execute_analysis(self, query_args):
+        # Hide the identity/version number of the server technology in the
+        # response headers.
+        cherrypy.response.headers["Server"] = "Tangelo"
```

Comment:
Good point, I'll change it to the empty string instead.

On Tue, Jan 19, 2016 at 9:29 AM David Manthey <notifications@github.com>
wrote:

> In tangelo/tangelo/server.py
> <https://github.com/Kitware/tangelo/pull/536#discussion_r50117518>:
>
> > @@ -541,6 +541,10 @@ def dirlisting(dirpath, reqpath):
> >          return result
> >
> >      def execute_analysis(self, query_args):
> > +        # Hide the identity/version number of the server technology in the
> > +        # response headers.
> > +        cherrypy.response.headers["Server"] = "Tangelo"
>
> Perhaps if security is the root cause of this, even reporting that this is
> Tangelo is too much.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/tangelo/pull/536/files#r50117518>.
>